src/pages/stacks-blockchain/overview.md

```diff
@@ -5,20 +5,20 @@ description: Learn more about the Stacks blockchain
 
 ## Introduction
 
-The Stacks 2.0 blockchain connects to Bitcoin for security and enables decentralized apps and predictable smart contracts. Stacks 2.0 implements a new mining mechanism called Proof of Transfer (PoX). PoX anchors to Bitcoin security and makes it possible to reward holders of Stacks (STX) tokens, the native token of the Stacks 2.0 blockchain, with bitcoins.
+Stacks 2.0 is a layer 1 blockchain that enables people to register digital assets and write smart contracts. It connects to Bitcoin with a 1:1 block ratio, meaning anything that happens on the Stacks blockchain can be verified on the Bitcoin Blockchain. It implements a new mining mechanism called Proof of Transfer (PoX). PoX uses the an established blockchain (in this case Bitcoin) to secure a new blockchain (Stacks).
```

Comment:
I wonder if `layer 1 blockchain` will be well understood for folks without specific domain knowledge?

Reply:
`PoX uses the an established` > `PoX uses an established`

Reply:
I feel good about layer 1 being accessible to most people who are reading through our docs, but will change if you feel strongly about it. I think it's important for differentiating us from the myriad of projects built on Ethereum.

Reply:
all good, let's go for it :)